src/Tests/Microsoft.NET.Sdk.BlazorWebAssembly.Tests/StaticWebAssetsBaselines/Publish_DoesNotGenerateManifestJson_IncludesJSModulesOnBlazorBootJsonManifest.Publish.files.json

```diff
@@ -195,6 +195,9 @@
   "${ProjectPath}\\bin\\Debug\\${Tfm}\\publish\\wwwroot\\_framework\\blazorwasm-minimal.dll.br",
   "${ProjectPath}\\bin\\Debug\\${Tfm}\\publish\\wwwroot\\_framework\\blazorwasm-minimal.dll.gz",
   "${ProjectPath}\\bin\\Debug\\${Tfm}\\publish\\wwwroot\\_framework\\blazorwasm-minimal.pdb.gz",
+  "${ProjectPath}\\bin\\Debug\\${Tfm}\\publish\\wwwroot\\_framework\\dotnet-crypto-worker.${RuntimeVersion}.[[hash]].js",
+  "${ProjectPath}\\bin\\Debug\\${Tfm}\\publish\\wwwroot\\_framework\\dotnet-crypto-worker.${RuntimeVersion}.[[hash]].js.br",
+  "${ProjectPath}\\bin\\Debug\\${Tfm}\\publish\\wwwroot\\_framework\\dotnet-crypto-worker.${RuntimeVersion}.[[hash]].js.gz",
```

Comment:
So it looks like we are producing new published content for 6.0 apps. Is the intention that people can just ignore it and we know for definite that it will never be used at runtime?

Reply:
We know it won't be used at runtime.

These files are only present in 7.0.